### DISCUSSION TITLE
신명희 1회차 블로그 피드백 요청드립니다!

### DISCUSSION THREAD

**Discussion by mh0223** [Orm-camp/blog-feedback#33] [동글동글 3기 피드백]
<a href="https://velog.io/@mh0223/%EC%98%A4%EB%A6%84%EC%BA%A0%ED%94%84-%EC%B2%AB-%ED%8C%80-%ED%94%84%EB%A1%9C%EC%A0%9D%ED%8A%B8-%ED%9A%8C%EA%B3%A0%EB%A1%9D">블로그 링크</a>

솔직한 피드백 부탁드립니다!!

**Comment by baaanjy** [comment]
개선사항이 구체적이라 다음 프로젝트 때에는 훨씬 잘해내실 것 같아요! 파이팅!

**Comment by Devkdy22** [comment]
공감가는 부분들이 있어서 시간 가는 줄 모르고 읽었습니다! 역할 분담이나 의견 조율 등 팀장 역할을 하시느라 힘드셨을 상황이 많으셨을 것 같은데 고생 많으셨어요! 읽으면서 저희 팀 팀장님의 노고를 다시 한번 느꼈습니다 🥲

**Comment by Orm-camp** [comment]
첫 회고글 작성하시느라 정말 고생 많으셨습니다! 😊

- 프로젝트 일정 (기간)이 조금 더 보기 쉽게 정리되어 있었더라면 어느 정도의 시간을 들여 작업한 프로젝트였는지 알기 쉬웠을 거 같아요.
- 프로젝트 개요 (이름, 사용 스택, 명희님께서 맡은 파트 등)이 간략하게 정리되어 있었으면 처음 보시는 분들도 어떤 상황이었을지 더 잘 이해할 수 있을 것 같아요.
- 이모지로 파트를 구분하셔서 읽기 편했어요. 내용상 구분되는 파트 (K / P / T)는 조금 더 간격을 넓혀서 작성해도 좋을 거 같아요!

저도 시간 가는 줄 모르고 집중해서 읽었습니다 ㅎㅎ 그만큼 느끼셨던 바를 글로 잘 옮기셨다고 생각해요. 정말 고생 많으셨어요!
TRY가 적용되어 있을 명희님의 다음 프로젝트도 매우 기대됩니다!

**Comment by hansol-woff** [comment]
단순히 문제점이나 힘들었던 것의 나열로 끝나는게 아니라 문제에 대한 구체적인 해결 방법이나 TRY 제시를 통한 실천 계획으로 마무리돼서 너무 좋았고 재미있게 읽었습니다. 고생 많으셨어요!

**Comment by oratio100** [comment]
회고글 잘 봤습니다. 내용이 자세해서 상황을 이해하고 말씀하시고자 하는 내용이 뭔지 이해하기 좋았어요.
특히 협의 내용의 시각화, 지속적인 상기 부분에서 많은 공감이 갔습니다.
명희님이 정리하신 TRY 파트는 저에게도 좋은 지침이 될 것 같습니다:)

**Comment by naseulgee** [comment]
우유부단함에 관한 부분에 많이 공감갔던 것 같습니다
특히 혼자 열심히 해왔지만 협의된 내용과 달랐을 때의 상황은 팀원으로서 저를 돌아보기도 했고, 팀장으로서 어떤 행동을 해야했나를 생각하게 되었습니다👍

**Comment by structified** [comment]
객관적인 사실들과 함께 당시의 감정까지 자세히 담아주셔서 글에 몰입할 수 있었습니다.

**Comment by mycreature** [comment]
문제점에 대한 TRY를 상세히 작성해서 좋았습니다. 팀장님 한테 너무 죄송할 따름입니다

**Reply by mh0223** [reply]
죄송하다니요,,, 민섭님 든든했습니다,, 그런말하지마세요 ㅠㅠㅠ

**Comment by kimjigyun** [comment]
글 써주신거 너무 잘 읽었습니다.  먼저든 생각은 조장을 맡으시면서 부딪히는 상황들이 많으셨을텐데 고생하셨다 싶고, 조장으로서도 노력해주셔서 프로젝트를 잘 마치신것 같다는 생각했습니다.

**Comment by ghost** [comment]
아직 초반의 스킬을 숙지한 인원들이모임과 동시에  
사공도 많고, 더좋은 아이디어를 내기위해 여러 인원의 의견이 많아 많이 휘청 였지만 
명희님이 중간에서 교통 정리도 하고 스스로 시간을 할애해서 전체적인 가닥을 잡아 
오히려 팀원들이 의지할  수 있었습니다. 

정말 고생 많으셨습니다.

**Reply by mh0223** [reply]
회고록이라 뭔가 아쉬운 점을 길게 기술하긴했지만 사실 좋은 점이 더 많았어요,,! 윤환님도 그동안 고생많으셨어요!
